ramlapi.go

```diff
@@ -82,31 +49,64 @@ func ResourceVerbs(resource *raml.Resource) map[string]map[string]string {
 			}
 		}
 	}
-	if resource.Post != nil {
-		verbs["POST"] = map[string]string{
-			"handler": resource.Post.DisplayName,
-		}
+
+	mappings := map[string]*raml.Method{
```

Comment:
Just looking at the use of the pointer here. I need to know more about this type.